### PR TITLE
Add SortCommand

### LINKED ISSUE #105: Sort command
sort by interview score (ascending or descending)

### PR DESCRIPTION
**Pull request by oolongwater**
Closes #105

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/SortCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    @Override
+    public SortCommand parse(String args) throws ParseException {
+        String trimmedArgs = args.trim();
+        if (trimmedArgs.equals("a")) {
+            return new SortCommand(true);
+        } else if (trimmedArgs.equals("d")) {
+            return new SortCommand(false);
+        } else {
+            throw new ParseException(String.format("Invalid sort order: %s. Use 'a' for ascending or "
+                    + "'d' for descending. Example: sort a", trimmedArgs));
```
Comment (SQ77) [inline src/main/java/seedu/address/logic/parser/SortCommandParser.java:25]: Should there be a check to see if no order is specified?
`sort` (with no order specified) will result in `Invalid sort order: . Use 'a' for ascending or 'd' for descending. Example: sort a`

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        List<Person> sortedList = model.getFilteredPersonList().stream()
+                .sorted((p1, p2) -> Float.compare(p1.getInterviewScore().toFloat(), p2.getInterviewScore().toFloat()))
+                .collect(Collectors.toList());
+
+        if (!isAscending) {
+            sortedList = sortedList.stream()
+                    .sorted((p1, p2) -> Float.compare(
+                            p2.getInterviewScore().toFloat(), p1.getInterviewScore().toFloat()))
+                    .collect(Collectors.toList());
+        }
```
Comment (SQ77) [inline src/main/java/seedu/address/logic/commands/SortCommand.java:52]: Is there a way to refactor this code to avoid sorting the list twice?

**Review by SQ77 — COMMENTED** [review]
Try executing this sequence of commands
`find David Bernice Alex` -> works
`sort a` -> works
`list` -> this command does not work
Seems like the sort command modifies the main list permanently?

**Review by bigismols — APPROVED** [review]
LGTM!

**Pull request merged**